python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -81,6 +81,10 @@ def __init__(
             ray_remote_args,
         )
         self._ray_remote_args = self._apply_default_remote_args(self._ray_remote_args)
+        self._ray_actor_task_remote_args = {}
+        actor_task_errors = DataContext.get_current().actor_task_retry_on_errors
+        if len(actor_task_errors) > 0:
+            self._ray_actor_task_remote_args["retry_exceptions"] = actor_task_errors
```

Comment:
IIRC `retry_exceptions` can either be True/False or a list of exception types in ray core. and defaults to false. 
Maybe let's keep the behavior same in data. 
also remember to update the comments in `DataContext`.

Reply:
yes, updated.